### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by gijzelaerr**

### CODE REVIEW

**Comment by gijzelaerr** [comment]
travis fails due to failing test, which seems to be unrelated to this PR. reported here https://github.com/ska-sa/tigger-lsm/issues/5

File: Tigger/SiameseInterface.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def compile_options (self):
     """Returns list of compile-time options""";
     if not self._compile_opts:
       self._compile_opts = [
-        TDLOption("filename","Tigger LSM file",
+        TDLRuntimeOptions("filename","Tigger LSM file",
```
Comment (o-smirnov) [inline Tigger/SiameseInterface.py]: Why did this become TDLRuntimeOptions? Needs to be a TDLOption still...

**Comment by o-smirnov** [comment]
See https://github.com/ska-sa/tigger-lsm/pull/4#pullrequestreview-117561539. The rest is unfathomable.

**Comment by o-smirnov** [comment]
Why is Travis failing?

**Comment by gijzelaerr** [comment]
fixed!

**Pull request merged**